### DISCUSSION TITLE
Run tests with dotnet test

### DISCUSSION THREAD

**Discussion by TenshiNet** [FlaUI/FlaUI#620] [Q&A]
Hello. I'm using FlaUI with xUnit. If I run the tests from Visual Studio, they work correctly, it launches the application and runs the tests. However, when I try to execute them from the console with dotnet test, it's unable to launch the application.

Someone else is experiencing this issue.

Thanks!!

**Comment by jbe2277** [comment]
I'm using FlaUI with xUnit too: https://github.com/jbe2277/waf/tree/master/src/Samples.UITest

I use a GitHub Action to run the UI tests automatically. For this I'm using `dotnet test`.

Extract from [System.Waf.CI.yml](https://github.com/jbe2277/waf/blob/master/.github/workflows/System.Waf.CI.yml):
```
dotnet test ./src/Samples.UITest/Samples.UITest.sln --logger "console;verbosity=detailed" -maxCpuCount:1
```
